### ISSUE TITLE
Name of the board not visible at WeKan v7.96

### ISSUE THREAD

**Issue by sunilnc** [wekan/wekan#5898]
I installed v7.96 yesterday, but I'm running into a series of regression issues when it comes to the UI:

Example 1: random blue outline and blue shading over the board names. I can't see the name of the board.
<img width="338" height="162" alt="Image" src="https://github.com/user-attachments/assets/f13e738c-12f9-40df-aa2c-acf1e46b6ddb" />

Example 2: blue box when creating list:
<img width="293" height="262" alt="Image" src="https://github.com/user-attachments/assets/537e465f-e628-4cf6-9d6d-158a781f8d33" />

Example 3: blue box when creating card:
<img width="308" height="279" alt="Image" src="https://github.com/user-attachments/assets/48174fe7-c8f6-4ef9-8728-b731f43da360" />

**Comment by xet7** [comment]
@sunilnc 
 
1. Looks like a bug
2. Looks like some accessibility fix https://github.com/wekan/wekan/pull/5895
3. Looks like some accessibility fix https://github.com/wekan/wekan/pull/5895

**Comment by xet7** [comment]
@sunilnc 

Have you set root url setting? Looks like too many of your WeKan UI is not translated, or you have not clicked language name.

```
sudo snap set wekan root-url='https://wekan.example.com'

export ROOT_URL=https://wekan.example.com
```

**Comment by sunilnc** [comment]
@xet7 I am not using snap wekan. I am using the regular instance of Wekan. How would I check?

**Comment by xet7** [comment]
@sunilnc 

What is "regular instance" ? Docker?

https://wekan.fi/install/

Docker:

https://github.com/wekan/wekan/blob/main/docker-compose.yml#L181

Bundle:

https://github.com/wekan/wekan/blob/main/start-wekan.bat#L17

**Comment by sunilnc** [comment]
@xet7 , yes sorry I mean I have installed via docker compose. 

What do in need to do?

**Comment by xet7** [comment]
@sunilnc 

https://github.com/wekan/wekan/blob/main/docker-compose.yml#L181

This should be your WeKan URL like https://wekan.example.com . And not localhost.

**Comment by xet7** [comment]
@sunilnc 

That is needed for translations and attachments to work correctly.

**Comment by sunilnc** [comment]
@xet7 this is my docker compose file:

```

services:
  wekan-db:
    image: mongo:5.0
    container_name: wekan-db
    restart: always
    volumes:
      - wekan-db:/data/db

  wekan:
    image: wekanteam/wekan
    container_name: wekan-app
    restart: always
    depends_on:
      - wekan-db
    ports:
      - "3001:8080"
    environment:
      - MONGO_URL=mongodb://wekan-db:27017/wekan
      - ROOT_URL=https://wekan.mydomain.co.uk:3001
      - MAIL_URL=smtp://myemail@gmail.com:myapppassword@smtp.gmail.com:587/?secure=true&ignoreTLS=false
      - MAIL_FROM=myemail@gmail.com
      - WITH_API=true
      - BROWSER_POLICY_ENABLED=true
      - BIGEVENTS_PATTERN=NONE
      - RICHER_CARD_COMMENT_EDITOR=false

volumes:
  wekan-db:

```

I've replaced mydomain with my actual domain and same with my email. Am I missing some extra configuration?

**Comment by xet7** [comment]
@sunilnc 

I don't know why you have port number `:3001` at ROOT_URL ? Does your webserver not proxy https 443 to 3001 ? Would it work without that port number at ROOT_URL ?

**Comment by xet7** [comment]
@sunilnc 

Webserver config examples are here:

https://github.com/wekan/wekan/tree/main/docs/Webserver

**Comment by sunilnc** [comment]
@xet7 fair point. I have removed the :3001 and restarted the container, and it still works. Did you want me to test something by making this change?

**Comment by xet7** [comment]
@sunilnc 

Do translations work? For example, at your screenshots above, should be `Add after list` and not untranslated `add-after-list` ?

Also, does it work to upload and download attachments at card?

**Comment by sunilnc** [comment]
@xet7 ah yes, that is now fixed:

<img width="321" height="201" alt="Image" src="https://github.com/user-attachments/assets/f3ffe4a6-7305-4539-a032-41696d27c17f" />

**Comment by xet7** [comment]
Fixed at https://github.com/wekan/wekan/commit/3814a218c2dab3ed9e8908c3a2de996ffebf6d9d . Thanks to xet7 !
